app/tests/retina_api_tests/helpers.py

```diff
@@ -290,10 +326,18 @@ def test_load_save_data(self, client):
         # login client
         client, grader = client_login(client, user="retina_user")
 
-        for archive in ("Rotterdam", "Australia"):
+        for archive in ("Rotterdam", "Australia", "kappadata"):
```

Comment:
Could you make the archives an enum? Would make usages of `kappadata` easier to handle.